apps/fluent-tester/src/TestComponents/StrokeWidth/StrokeWidthTest.tsx

```diff
@@ -7,18 +7,16 @@ import { stackStyle } from '../Common/styles';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { Text } from '@fluentui/react-native';
 import { Theme, useFluentTheme } from '@fluentui-react-native/framework';
-import { getCurrentAppearance } from '@fluentui-react-native/theming-utils';
 import { themedStyleSheet } from '@fluentui-react-native/themed-stylesheet';
 
 const getThemedStyles = themedStyleSheet((t: Theme) => {
-  const isLightMode = getCurrentAppearance(t.host.appearance, 'light') === 'light';
   return {
     root: {
       paddingBottom: 16,
     },
     exampleStrokeBackground: {
       height: 100,
-      backgroundColor: isLightMode ? globalTokens.color.grey80 : globalTokens.color.grey10,
+      backgroundColor: t.colors.bodyStandoutBackground,
```

Comment:
Sorry @lyzhan7 but this isn't an alias token 😓 this is an older color that we'll want to remove at some point. You should take a color from [fluentui-react-native/Color.types.ts at f14e559f197dc2c1b7631de0100aa30742648544 · microsoft/fluentui-react-native (github.com)](https://github.com/microsoft/fluentui-react-native/blob/f14e559f197dc2c1b7631de0100aa30742648544/packages/theming/theme-types/src/Color.types.ts#L453)

Reply:
Same with bodyText

Reply:
Ah okay, I don't think any of the aliascolortokens exist on iOS (yet). I can either just update this right now to use one of those tokens (and have a temporary special case for iOS using the globalToken values I had), or just use globalTokens for now and come back to this after I finish the color token work for ios

Reply:
Yeah you can use globals for now that's fine. Thanks for working through my comments!

Reply:
Okay, I'll go back to global tokens for now. Made an ADO item to come back to these tester pages as part of the iOS color token work